### ISSUE TITLE
CleverTap users are unable to migrate to v2.0

### ISSUE THREAD

**Issue by piyush-kukadiya** [segmentio/analytics-react-native#582]
## Summary

@alanjcharles Our users are unable to use v2.0 with `analytics-react-native-clevertap`. How can we proceed from here? Any ETA for supporting `analytics-react-native-clevertap` with 2.0. Any help needed from CleverTap team in terms of coding a plugin?

Thanks.

**Comment by alanjcharles** [comment]
@piyush-kukadiya we can try to prioritize adding a Clevertap Plugin this sprint! In the meantime, please feel free to build one that works for you. [You can follow this documentation](https://github.com/segmentio/analytics-react-native)

**Comment by ShubhamBathe-KM** [comment]
@alanjcharles Any updates regarding CleverTap plugin?

**Comment by oscb** [comment]
Alan is out this week, but we're working on this. I don't have yet an ETA but it's our first priority at the moment.

**Comment by ShubhamBathe-KM** [comment]
Thanks for the update @oscb.

**Comment by piyush-kukadiya** [comment]
@oscb User's apps are getting rejected by play store due to non compliant sdk version`com.segment.analytics.android:analytics:4.6.0` and the sdk must be migrated to **4.10.1**.  

`@segment/analytics-react-native` **1.5.0** is pointing to `com.segment.analytics.android:analytics:4.9.3`. Can you please make it possible to release newer version of `analytics-react-native-clevertap `with **2.0** asap. Also any workaround as of now that we can share with users? What if Apps make changes in `/node_modules/@segment/analytics-react-native/android/build.gradle` to point sdk to **4.10.1**? Will `@segment/analytics-react-native` **1.5.0** will be compatible with `com.segment.analytics.android:analytics:4.10.1`? Thanks!

**Comment by oscb** [comment]
@piyush-kukadiya That has been already fixed in v1.5.2 https://www.npmjs.com/package/@segment/analytics-react-native/v/1.5.2

Please use that version until we can release the CleverTap plugin.

**Comment by piyush-kukadiya** [comment]
Thanks! @oscb

**Comment by minhchienwikipedia** [comment]
@oscb do you guys have a new version for CleverTap?

**Comment by piyush-kukadiya** [comment]
@oscb @alanjcharles Any updates here? Can we have rough estimates for this. We have users waiting on this for long. Thanks!

**Comment by kathaypacific** [comment]
hi @oscb @alanjcharles @piyush-kukadiya are there any updates on when the clevertap plugin for v2 will be ready? we've also been waiting for this and blocked from upgrading to v2. thanks!

**Comment by oscb** [comment]
@kathaypacific no updates to share yet on this. 
It's still in our roadmap before deprecating v1 but we've been blocked to actually get this ready.

**Comment by alanjcharles** [comment]
@kathaypacific @piyush-kukadiya @ShubhamBathe-KM  CleverTap is now available for Analytics React Native 2.0. You can find it here: https://www.npmjs.com/package/@segment/analytics-react-native-plugin-clevertap 

Thank you for your patience. Please feel free to open a new issue if you run into any problems with the integration.
